ui/src/reducers/remoteAccess.js

```diff
@@ -36,8 +36,11 @@ function remoteAccessReducer(state = INITIAL_STATE, action = {}) {
     case 'RESET_CHAT_MESSAGE_COUNT': {
       return { ...state, chatMessageCount: 0 };
     }
-    case 'INC_CHAT_MESSAGE_COUNT': {
-      return { ...state, chatMessageCount: state.chatMessageCount + 1 };
+    case 'SET_CHAT_MESSAGE_COUNT': {
```

Comment:
Simply made it possible to pass the number of unread messages

Reply:
The naming doesn't make it clear that the message count is being incremented. I think the `INC` prefix was fine before; "to increment" doesn't necessarily mean by 1.

Reply:
I see,
 
> "to increment" doesn't necessarily mean by 1.

True, I somehow found it nicer with SET but INC is just as fine